test/test-cases/regression/request-body-parser-multipart.json

```diff
@@ -2448,6 +2448,452 @@
         "SecRule FILES_SIZES:/^image/ \"@eq 0\" \"phase:2,deny,id:500167\"",
         "SecRule &FILES_TMPNAMES \"!@eq 2\" \"phase:2,deny,id:500168\""
     ]
+  },

```

Comment:
Thank you for providing the test case altogether with the change.

Reply:
You're welcome - I'm enjoying to participate :)